src/react-mf-navbar.js

```diff
@@ -7,9 +7,20 @@ import Root from "./root.component";
 const lifecycles = singleSpaReact({
   React,
   ReactDOM,
-  rootComponent: Root
+  rootComponent: Root,
+  domElementGetter
 });
 
 export const bootstrap = lifecycles.bootstrap;
 export const mount = lifecycles.mount;
 export const unmount = lifecycles.unmount;
+
+function domElementGetter() {
+  let el = document.getElementById("navbar");
+  if (!el) {
+    el = document.createElement("nav");
```

Comment:
Doing it this way allows me to set the dom node to be `<nav>` instead of `<div>` plus I think it's a good example